Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -298,44 +281,35 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
     private let controllerErrorStore: NetworkProtectionTunnelErrorStore
     private let latencyReporter = NetworkProtectionLatencyReporter(log: .networkProtection)
 
-    // MARK: - Notifications: Observation Tokens
-
-    private var requestStatusUpdateCancellable: AnyCancellable!
-
     // MARK: - Cancellables
 
     private var cancellables = Set<AnyCancellable>()
 
     // MARK: - Initializers
 
-    private let notificationCenter: NetworkProtectionNotificationCenter
     private let useSystemKeychain: Bool
     private let debugEvents: EventMapping<NetworkProtectionError>?
     private let providerEvents: EventMapping<Event>
     private let appLauncher: AppLaunching?
 
-    public init(notificationCenter: NetworkProtectionNotificationCenter,
-                notificationsPresenter: NetworkProtectionNotificationsPresenter,
+    public init(notificationsPresenter: NetworkProtectionNotificationsPresenter,
+                tunnelHealthStore: NetworkProtectionTunnelHealthStore,
+                controllerErrorStore: NetworkProtectionTunnelErrorStore,
```

Comment:
These changes should be fine because they make this class more testable in theory.